app/com/lynxanalytics/biggraph/controllers/ProtoTable.scala

```diff
@@ -38,11 +42,7 @@ trait ProtoTable {
     // unique to the ProtoTable. This breaks a couple of things down the line, so we have to strip
     // it out when this relation is used anywhere else, which might not be trivial.
     val fields = schema.fields.map(f => f.withComment(s"$this"))
-    if (fields.nonEmpty) {
-      LocalRelation(fields.head, fields.tail: _*)
-    } else {
-      LocalRelation()
-    }
+    new ProtoLocalRelation(this, fields)
```

Comment:
This is probably a better solution than the comment. But I didn't dare mess with this scary code. Should I?

Reply:
Just for the record. We concluded it would most probably work to stop using comments altogether and take the proto tables directly from the `ProtoLocalRelation`s. But we are not 100% sure that it's impossible that in some cases the ProtoLocalRelation get converted into some other kind of `LogicalPlan` in which case we lose this information.